frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -51,12 +50,26 @@ const StyledTypography = styled(Typography)`
     font-style: normal;
     font-weight: 400;
     line-height: 40px; /* 125% */
-
     @media (max-width: 500px) {
         font-size: 24px;
     }
 `
 
+const StyledMissionHeader = styled(Card)`
+    display: flex;
+    padding: 10px 10px 10px 10px;
```

Comment:
Wouldn't this be the same as just saying padding: 10px?
```suggestion
    padding: 10px;
```